modules/integration_aws-lambda/conf/01-errors-percentage.yaml

```diff
@@ -3,19 +3,19 @@ name: errors percentage
 id: pct_errors
 
 transformation: true
-aggregation: true
-filtering: "filter('namespace', 'AWS/Lambda') and filter('stat', 'mean') and filter('Resource', '*')"
+aggregation: ".mean(by=['FunctionName'])"
+filtering: "filter('namespace', 'AWS/Lambda') and filter('stat', 'sum') and filter('Resource', '*')"
 value_unit: "%"
```

Comment:
I still desigree about the necessity for aws metrics, we have lot of orgs with cloudwatch metrics with bigger interval (5m) than lambda service (1m) AND greater interval for cloudwatch integration like 10m which do not suffer of this problem.
I am sure there is a specific problem in your case to explain such a big delay whereas you configured the clouwatch integration with the minimum 60 sec interval.
However, it can at least be useful as workaround for this kind of cases so I created an issue: https://github.com/claranet/terraform-signalfx-detectors/issues/375 I will implement in future